### ISSUE TITLE
Tranlslate Polkadot A to Z: A for Accounts into Hausa

### ISSUE THREAD

**Issue by emresurmeli** [w3f/1KC#11]
Original post:
https://www.reddit.com/r/Polkadot/comments/o7qa6v/a_for_account_polkadot_from_a_to_z/

Task details:
To translate the Polkadot A to Z: A for Accounts blog post to Hausa. The reward will be calculated at 10 cents/word, for a total of 393 words equaling to a $39.3 reward. Payout format will be up to the Tech Ed team, in the form of a stable coin of their choice. The translation work will need to be QA'ed internally, the team may request changes from the original translator, and reserve the right to deny and work that does not meet minimum standards based on discretion of the reviewing team/s.

Task assigned to:
@abujulaibib

Working document:
https://docs.google.com/document/d/1_3gkVJATYWvivGEnrf9-fzdPMUotfaTRWbqd4k6QDd4/edit?usp=sharing

**Comment by abujulaibib** [comment]
@emresurmeli hey, I am through with the translation

**Comment by abujulaibib** [comment]
I have an observation. This series suppose to follow an A to Z format, with every letter assigned corresponding term in the ecosystem. For English this is perfectly doable since the whole series is started in English, but translating into another language may pose a problem as the terms may not necessarily tally with the corresponding alphabet. Should we leave the English terms as they are and proceed with translating the body alone or must the terms also be translated?

**Comment by emresurmeli** [comment]
@abujulaibib Good point, I think this is fine to keep the English as well as the translated title, I'm working on how to post the translated versions and once we decide I am planning on Keeping the English title, but also translating the title in smaller text right under it.

**Comment by abujulaibib** [comment]
Ok, that will be a good approach
